.github/workflows/docker-images.yml

```diff
@@ -32,14 +32,15 @@ on:
         default: false
       omes-repo-path:
         type: string
-        default: 'temporalio/omes'
+        default: "temporalio/omes"
       omes-repo-ref:
         type: string
 
 jobs:
   publish-artifacts:
     name: Publish ${{ inputs.lang }} artifacts
     runs-on: ubuntu-latest
+    timeout-minutes: 20
```

Comment:
I saw a [runaway job](https://github.com/temporalio/omes/actions/runs/17308616687/job/49144094871?pr=171) that still wasn't done after 2h+